sky/packages/sky/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,114 @@
+import 'dart:ui' as ui;
```

Comment:
Hm, the original intent was to make them mutable, but now that I think about it that's not really necessary - we just needed to be able to assign the pointer value away from what the native layer's value was. I think I should make all the fields here final, do you agree?